docs/source/tutorials/text.ipynb

```diff
@@ -687,7 +531,7 @@
    "source": [
     "# Note: This cell is only for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
     "\n",
-    "highlighted_indices = [5204, 22294, 15079]  # check these examples were found in find_label_issues\n",
+    "highlighted_indices = [646, 390, 628, 702]  # check these examples were found in find_label_issues\n",
```

Comment:
just check these one last time to verify they are the only examples ever mentioned / highlighted in the notebooks

Reply:
Verified!